open-api/rest-catalog-open-api.yaml

```diff
@@ -3804,6 +4110,41 @@ components:
             }
           }
 
+    #  Note that this is a representative example response for use as a shorthand in the spec.
+    #  The fields `message` and `type` as indicated here are not presently prescriptive.
+    MisdirectedRequestResponse:
```

Comment:
@rdblue I think you suggested 422 for this behavior, but since we now use 422 for response too big case, I feel 421 is pretty suitable. This is too some extent a "misdirected request" since the client is supposed to call preplan first (based on information like server capability). Let me know if you agree if 421 is suitable.

Reply:
+1